bolero/optimizer/test/test_optimizer.py

```diff
@@ -2,14 +2,27 @@
 from bolero.utils.testing import all_subclasses
 from bolero.optimizer import Optimizer, ContextualOptimizer
 from bolero.utils.testing import assert_pickle
-from nose.tools import assert_false, assert_true, assert_equal
+from nose.tools import (assert_false, assert_true, assert_equal,
+                        assert_raises_regexp)
 from numpy.testing import assert_array_equal
 
 
 ALL_OPTIMIZERS = all_subclasses(Optimizer)
 ALL_CONTEXTUALOPTIMIZERS = all_subclasses(ContextualOptimizer)
```

Comment:
all_subclasses returns an empty list.. So some tests actually do not run.

Reply:
Uh, didn't see that before. I introduced this bug here: https://github.com/rock-learning/bolero/commit/abd258d2e61fc635b54ac4eec5848308f8acf11b#diff-39d53a7dd0adc4b9b1e549b967aa6034

Reply:
Oops, unfortunate commit, but easy to fix then, if its just this code indentation ;)